src/responder/nss/nss_protocol_grent.c

```diff
@@ -402,9 +402,14 @@ sss_nss_protocol_fill_initgr(struct sss_nss_ctx *nss_ctx,
     ret = sysdb_search_group_by_origgid(NULL, domain, orig_gid, NULL,
                                         &primary_group_msg);
     if (ret != EOK) {
-        DEBUG((ret == ENOENT ? SSSDBG_FUNC_DATA : SSSDBG_MINOR_FAILURE),
-              "Unable to find primary gid [%d]: %s\n",
-              ret, sss_strerror(ret));
+        if (ret == ENOENT) {
+            DEBUG(SSSDBG_FUNC_DATA, "There is no override for this group\n");
```

Comment:
Let's include `orig_gid` value in the message?

Reply:
Done.